package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "eslint-config-netflix",
+  "name": "@netflix/eslint-config-netflix",
```

Comment:
Looks like the scope prefix wasn’t previously added. Just committing the result of running `npm install` here.